### PR TITLE
Ugh. Bad rebase.

### PR DESCRIPTION
**Pull request by mattsp1290**
Copy-paste to fix previous copy-paste

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
+{
```
Comment (animate-object) [inline package-lock.json:1]: excellent, it will be good to have both a `package-lock.json` and a `yarn.lock`

File: package.json [hunk 2]
```diff
@@ -31,6 +31,8 @@
   },
   "dependencies": {
     "is-is-odd": "^1.0.2",
-    "is-odd": "^3.0.1"
+    "is-odd": "^3.0.1",
+    "jquery": "^3.4.1",
+    "jquery-basic-arithmetic-plugin": "^1.1.0"
   }
```
Comment (animate-object) [inline package.json:40]: don't forget to add your name to the contributors field, so you are forever associated with this project.

File: index.js [hunk 3]
```diff
@@ -25,5 +27,5 @@ module.exports = function(val) {
 }
 
 const _isTen = (val) => {
-  return val === 10;
+  return jQuery.equals(val, 10);
```
Comment (animate-object) [inline index.js:30]:
```suggestion
  return jQuery.equals(jQuery.divide(val, jQuery.ten), jQuery.one);
```

https://github.com/cbrandolino/jQuery-basic-arithmetic-plugin/blob/master/jquery.basic_arithmetics.js#L94

**Review by animate-object — CHANGES_REQUESTED** [review]
looks like a great start! please address pull request comments and we can get this through

**Comment by mattsp1290** [comment]
@animate-object  Made changes requested. Not sure why, but jQuery.ten and jQuery.one are undefined.

**Review by animate-object — APPROVED** [review]
really nice work on this. it does what it did before, but with jquery. wow!

**Pull request merged**